### ISSUE TITLE
Error: Couldn't resolve the package 'os_detect'

### ISSUE THREAD

**Issue by thorizer** [dart-lang/os_detect#19]
i get this on windows and web

**Comment by lrhn** [comment]
Did you add a dependency in your `pubspec.yaml` and run `dart pub get`?

**Comment by thorizer** [comment]
@lrhn  yes, actually it happens with some packages like this one, cause i added the dependency on the repository package not the main one.
